prometheus/src/main/java/org/opensearch/sql/prometheus/request/system/PrometheusDescribeMetricRequest.java

```diff
@@ -94,16 +94,16 @@ public List<ExprValue> search() {
     List<ExprValue> results = new ArrayList<>();
     for (Map.Entry<String, ExprType> entry : getFieldTypes().entrySet()) {
       results.add(row(entry.getKey(), entry.getValue().legacyTypeName().toLowerCase(),
-          catalogSchemaName));
+          dataSourceSchemaName));
     }
     return results;
   }
 
   private ExprTupleValue row(String fieldName, String fieldType,
-                             CatalogSchemaName catalogSchemaName) {
+                             DataSourceSchemaName dataSourceSchemaName) {
     LinkedHashMap<String, ExprValue> valueMap = new LinkedHashMap<>();
-    valueMap.put("TABLE_CATALOG", stringValue(catalogSchemaName.getCatalogName()));
-    valueMap.put("TABLE_SCHEMA", stringValue(catalogSchemaName.getSchemaName()));
+    valueMap.put("TABLE_CATALOG", stringValue(dataSourceSchemaName.getDataSourceName()));
```

Comment:
is this table_catalog notion differs from the datasource ?

Reply:
both are same.

TABLE_CATALOG  is pre-existing response column when we query metadata about opensearch indices.
I am following the same convention right now in order to be consistent with opensearch metadata queries.
Need to re-think if we can remove from both the places.

Will do it in a seperate PR because we need to think about backward compatibility of existing queries.

May be we can change TABLE_CATALOG to TABLE_DATASOURCE in prometheus.